module/TaskManagement/test/integration/TaskManagement/TaskAcceptanceTest.php

```diff
@@ -0,0 +1,132 @@
+        'paul.smith@ora.local'  => 'eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXUyJ9.eyJ1aWQiOiIyMDAwMDAwMC0wMDAwLTAwMDAtMDAwMC0wMDAwMDAwMDAwMDAiLCJpYXQiOiIxNDM4NzgzMjQzIn0.WTKW0CBmHlHIfBmtzTeakDUlX0p775w59bT1FKN2TIYcJ3nBEF_hmY0s3eEKZ6dOs4PjxyskVRYiB5dlbG1ZSYRbOJGysn5lvltXBmhOk2Ad3RiI8rina-Af0eBXS96A2BY2Qc2NN5t3EcjmIateH_dgG85adewQSZVJTTKKUBid46fdZ0TO5Y1jcr153xxMuE66W9gMGP2ffUGJIt01UQeuljQM1OF8Ss87l9tIcgRrKd5NiU5ap6JY4nTiZYgh8d7LPd4NfZ34GdQjt0vM0J9q_pQ9dN2GzuF9MO09TjRfmMNuE-fHboye4ahTaHH2OcUFDEMOF6XWy8tw8t7E3A',
+        'bruce.wayne@ora.local' => 'eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXUyJ9.eyJ1aWQiOiI4MDAwMDAwMC0wMDAwLTAwMDAtMDAwMC0wMDAwMDAwMDAwMDAiLCJpYXQiOiIxNDM4NzgzMzE0In0.PFaRVhV_us6hLMjCyfVcA1GdhoSDlZDInOa-g7Ks2HMLYqiaOwzoRjxhLObBY8KQZ4h9mkBbhycnO6HsX6QtXlxdqB4jGACGAQzGxfS9l4kIUJzHacQxVO0SW58U-XITpKZL6tAnLo_rpfnWFdTKUWZ1lBx0Z7ymPiHIqmlrBSdXW9JJTP4OVCq4CsxfUpT65DcLCJebJ7rDbMgCGy6C2SvP676IjBqKeAf44_XjolvBvqHWbYx6WrgbQfZQpPmaqhggyKRRcivgsp8bd1GOuxM9bvXRagdqF1suac5SXZG8vgv-V3UjxyZpmu7XsJeWO085pPsOvG3i7EvIRKgqbg',
+    ];
+
+    public function setUp()
+    {
+        $config = getenv('APP_ROOT_DIR') . '/config/application.test.config.php';
+        $this->client = ZFHttpClient::create($config);
+        $this->client->enableErrorTrace();
+        $this->client->setJWTToken(static::$tokens['bruce.wayne@ora.local']);
+    }
+
+
+	public function testReopenTask()
+    {
+        $serviceManager = $this->client->getServiceManager();
+        $userService = $serviceManager->get('Application\UserService');
+        $admin = $userService->findUserByEmail('bruce.wayne@ora.local');
+        $member = $userService->findUserByEmail('phil.toledo@ora.local');
+
+        $res = $this->createOrganization($serviceManager,'my org', $admin, [], [$member]);
+        $task = $this->createTask($serviceManager, 'Lorem Ipsum Sic Dolor Amit', $res['stream'], $admin, [$member]);
+
+        $response = $this->client
+            ->get("/{$res['org']->getId()}/task-management/tasks/{$task->getId()}");
+        $data = json_decode($response->getContent(), true);
+
+        $this->assertEquals(TaskInterface::STATUS_COMPLETED, $data['status']);
+        $this->assertCount(0, $data['acceptances']);
+
+        $acceptanceData = [
+            'value' => Vote::VOTE_AGAINST,
+            'description' => 'no money for you!'
+        ];
+        $response = $this->client
+                         ->post(
+                             "/{$res['org']->getId()}/task-management/tasks/{$task->getId()}/acceptances",
+                             $acceptanceData
+                         );
+
+        $this->assertEquals('201', $response->getStatusCode());
+
+
+        $data = json_decode($response->getContent(), true);
+
+        $this->assertEquals(TaskInterface::STATUS_ONGOING, $data['status']);
+        $this->assertCount(0, $data['acceptances']);
+	}
+
+	protected function createOrganization($serviceManager, $name, $admin, array $members = [], array $contributors = [])
```

Comment:
non avevamo una classe helper che conteneva questo metodo?

Reply:
Yes, si trova in un vecchio ramo non ancora ricondotto a master.
Faccio il merge e procedo ad un refactoring